test/data/pci_invalid.tf

```diff
@@ -0,0 +1,36 @@
+provider "instaclustr" {
```

Comment:
This example isn't invalid just for PCI. Copy configuration from pci_valid.tf and toggle an option to invalidate it instead (for example private_network_cluster = false)